configs/libcall.toml

```diff
@@ -0,0 +1,67 @@
+# This example configuration covers detailed configuration for each sampler
```

Comment:
Maybe change this text to talk about this particular sampler you're showing off?

Reply:
I shouldn't have left that boilerplate in there. I think the sample config covers most of this so I just cleaned things up a bit.